Wordle.Api/Wordle.Api/Models/AppUser.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Identity;
+
+namespace Wordle.Api.Models;
+public class AppUser : IdentityUser
+{
+    public int GameCount { get; set; }
+    public double AverageAttempts { get; set; }
+    public int AverageSecondsPerGame { get; set; }
+
+    public DateOnly BirthDate { get; set; }
+    public bool IsMasterOfTheUniverse { get; set; }
```

Comment:
This isn't really a big deal since it seems everyone else did it this way, but why even have a MasterOfTheUniverse claim if each AppUser object just has a bool that could be checked? Unless I've completely misunderstood something and my site is just magically working (which is possible), I don't think you need a backing variable like this for claims. Claims and roles are stored with Azure's identity systems already, no need for a bool here in your own database.
My gut tells me it might even be a security issue if more important "claims" were saved in this way in a real service.

Reply:
You might be right. I just did it that way because I don't fully understand how claims work. Thank you for your explanation.